<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Puzzle_2020_24_Display</class>
 <widget class="QWidget" name="Puzzle_2020_24_Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Island Resort Lobby</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="m_push_button_advance">
       <property name="text">
        <string>Advance</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QSpinBox" name="m_spin_box_nb_days">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="m_label_days">
       <property name="text">
        <string>day</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_label_day">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Day 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="m_check_box_show_advancement">
       <property name="text">
        <string>Show advancement</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Puzzle_2020_24_View" name="m_view" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Puzzle_2020_24_View</class>
   <extends>QWidget</extends>
   <header>solvers/2020/display/puzzle_2020_24/puzzle_2020_24_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
